CONTRIBUTING.md

```diff
@@ -61,6 +61,19 @@ tabs). Format your code assuming it will be read in a window 100
 columns wide. Wrap code at 100 characters and comments at 80 unless doing so
 makes the code less legible.
 
+When wrapping function signatures that do not fit on one line,
+put the name, arguments and return types on separate lines, with the closing `)`
+at the same indentation as `func`. Example:
+```go
+func (s *someType) myFunctionName(
+    arg1 somepackage.SomeArgType, arg2 int, arg3 somepackage.SomeOtherType,
+) (somepackage.SomeReturnType, error) {
+    ...
+}
+```
+If the arguments list exceeds 100 characters, switch to one argument per line. 
```

Comment:
I think this might read more easily if you included examples for all the cases:
- everything on one line
- args on separate line
- returns on separate line (maybe omit this one - I think cases that want this can just use args on separate line)
- one-arg-per-line
- one-return-per-line

also avoid restating "100 characters" here, it's already in the same doc. perhaps put an anchor on the char limit and link to it from here.